src/app/tests/suites/certification/Test_TC_DGTHREAD_1_1.yaml

```diff
@@ -54,7 +54,7 @@ tests:
     - label:
           "Given DGTHREAD.S.F00(PKTCNT) ensure featuremap has the correct bit
           set"
-      PICS: DGETH.S.F00
+      PICS: DGTHREAD.S.F00
```

Comment:
`DGTHREAD.S.F00` does not exist in PICS v0.11.

Reply:
@yinyihu-silabs testplan tells to use PICS DGTHREAD.S.F00 
![image](https://user-images.githubusercontent.com/102359958/188701122-cc9d0731-f3ab-4ee2-af9e-eb0cb66562ed.png)